v2/tools/asoctl/cmd/import_azure_resource.go

```diff
@@ -11,13 +11,11 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-
+	"github.com/Azure/azure-service-operator/v2/api"
 	"github.com/Azure/azure-service-operator/v2/internal/genericarmclient"
 	"github.com/Azure/azure-service-operator/v2/internal/version"
-
-	"github.com/Azure/azure-service-operator/v2/api"
+	"github.com/pkg/errors"
```

Comment:
minor: I think the old import ordering was more correct, as the ASO stuff should be at the bottom and the GH stuff should be in the middle.